Fix build of Trinity plugin bamsifter:
* use external HTSlib from EasyBuild environment
* use compilation flags from environment
author: Alex Domingo (Vrije Universiteit Brussel)
diff -ru trinityrnaseq-v2.15.2.orig/trinity-plugins/bamsifter/Makefile trinityrnaseq-v2.15.2/trinity-plugins/bamsifter/Makefile
--- trinityrnaseq-v2.15.2.orig/trinity-plugins/bamsifter/Makefile	2024-08-01 14:53:23.000000000 +0200
+++ trinityrnaseq-v2.15.2/trinity-plugins/bamsifter/Makefile	2024-10-21 07:50:10.911559350 +0200
@@ -2,8 +2,8 @@
 
 cwd = $(shell pwd)
 
-sift_bam_max_cov: sift_bam_max_cov.cpp htslib/version.h
-	g++ -std=c++11 -o _sift_bam_max_cov sift_bam_max_cov.cpp -Wall -O2 -L./htslib/build/lib/ -I./htslib/build/include -lhts
+sift_bam_max_cov: sift_bam_max_cov.cpp
+	g++ -std=c++11 $(CXXFLAGS) -Wall -I$(EBROOTHTSLIB)/include -L$(EBROOTHTSLIB)/lib -lhts -o _sift_bam_max_cov sift_bam_max_cov.cpp
 
 
 htslib/version.h :
